File: Java/FJ - 11/6.8 fj11-contas/src/br/com/gft/model/Conta.java
package br.com.gft.model;

public class Conta {
	private double saldo;
	private String titular;
	private int numero;
	private String agencia;

	public void setTitular(String titular) {
		this.titular = titular;
	}
	
	public boolean deposita(double qtd) {
		if(qtd < 0) {
			return false;
		}
		else {
			this.saldo += qtd;
			return true;
		}
	}
	
	public boolean saca(double qtd) {
		if(qtd < 0) {
			return false;
		}
		else {
			this.saldo -= qtd;
			return true;
		}
	}
}

File: C#/FN -13/21.7 LINQ/21.7 LINQ/Conta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _21._7_LINQ
{
    public class Conta
    {
        public Conta(string titular, int numero, int agencia)
        {
            Saldo = 0;
            Titular = titular;
            Numero = numero;
            Agencia = agencia;
        }

        public double Saldo { get; private set; }
        public string Titular { get; }
        public int Numero { get; }
        public int Agencia { get; }

        public override string ToString()
        {
            return string.Format("Titular: {0}, Saldo: {1}", Titular, Saldo);
        }

        public void Saca(double qtd)
        {
            Saldo -= qtd;
        }
        public void Deposita(double qtd)
        {
            Saldo += qtd;
        }
    }
}
